torchsde/_core/methods/heun.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
Could we add a docstring like [this](https://github.com/google-research/torchsde/blob/381f4e9854b275faa5629915c1f805bf78090dd8/torchsde/_core/methods/srk.py#L15) describing the full name of the scheme and which paper the scheme comes from?

Reply:
Sure! Added one suggested below.

Reply:
Preferably specify it slightly more specifically than "Strong order 1.0 scheme" because that's quite open-ended ;)

Reply:
Sure, could it be: "Stratonovich Heun method (strong order 1.0 scheme) from..."? (such name is used in the cited paper)

Reply:
Yep, that sounds good to me.

Reply:
Done!